packages/xwayland.rb

```diff
@@ -151,6 +151,6 @@ def self.install
     system "DESTDIR=#{CREW_DEST_DIR} ninja -C build install"
     FileUtils.mv "#{CREW_DEST_PREFIX}/bin/Xwayland", "#{CREW_DEST_PREFIX}/bin/Xwayland.elf"
     system "install -Dm755 Xwayland_sh #{CREW_DEST_PREFIX}/bin/Xwayland"
-    FileUtils.ln_sf "#{CREW_DEST_PREFIX}/bin/Xwayland", "#{CREW_DEST_PREFIX}/bin/X"
+    system "ln -sfr #{CREW_DEST_PREFIX}/bin/Xwayland #{CREW_DEST_PREFIX}/bin/X"
```

Comment:
Actually, it should be:
```
FileUtils.ln_sf "#{CREW_PREFIX}/bin/Xwayland", "#{CREW_DEST_PREFIX}/bin/X"
```
In the future, please test before submitting a PR.  It will save me time. :)

Reply:
Touché.